### PR TITLE
CoreOS guide for GCE

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: docs/cni/kubernetes/cloud-config/node-config.yaml [hunk 1]
```diff
@@ -1,6 +1,17 @@
 #cloud-config
 ---
 write_files:
+  - path: /etc/cni/net.d/10-calico.conf
```
Comment (tomdee) [inline docs/cni/kubernetes/cloud-config/node-config.yaml]: This path looks wrong

File: docs/cni/kubernetes/GCE.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying Calico and Kubernetes on GCE 
+
+These instructions allow you to set up a Kubernetes v1.1.3 cluster with [Calico networking][calico-networking] on GCE using the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components.
```
Comment (tomdee) [inline docs/cni/kubernetes/GCE.md]: Broken link

File: docs/cni/kubernetes/GCE.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config/master-config.yaml
+```
+
+Deploy at least one worker node using the following command:
+```
+gcloud compute instances create \
+  kubernetes-node-1 \
+  --image-project coreos-cloud \
+  --image coreos-alpha-709-0-0-v20150611 \
```
Comment (tomdee) [inline docs/cni/kubernetes/GCE.md]: That's a _very_ old image

File: docs/cni/kubernetes/GCE.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  --metadata-from-file user-data=cloud-config/node-config.yaml
+```
+
+## 3. Using your cluster
+### 3.1 Configuring kubectl
+The following steps configure remote kubectl access to your cluster.
+
+Download `kubectl`
+```
+wget https://storage.googleapis.com/kubernetes-release/release/v1.1.3/bin/linux/amd64/kubectl
```
Comment (tomdee) [inline docs/cni/kubernetes/GCE.md]: You need chmod too

File: docs/cni/kubernetes/GCE.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+```
+
+Get the IP address of your kubernetes-master
+```
+gcloud compute instances describe kubernetes-master | grep networkIP
```
Comment (tomdee) [inline docs/cni/kubernetes/GCE.md]: Isn't this appraoch a bit dangerous? You're opening up unsecured access to the kubenetes cluster to the world?

**Comment by tomdee** [comment]
Finished looking at this - there are a few comments to address then I'll take another look.

File: docs/cni/kubernetes/GCE.md (hunk 5 above)
Comment (caseydavenport) [inline docs/cni/kubernetes/GCE.md]: Hmm.. good point.  It should probably be secured, but didn't want to go through the trouble of TLS for this guide.  Maybe there is another way.

File: docs/cni/kubernetes/cloud-config/node-config.yaml (hunk 1 above)
Comment (caseydavenport) [inline docs/cni/kubernetes/cloud-config/node-config.yaml]: What looks wrong about it?

Comment (caseydavenport) [inline docs/cni/kubernetes/cloud-config/node-config.yaml]: Oh I see.

**Comment by caseydavenport** [comment]
Updated, ready for another look @tomdee

**Comment by tomdee** [comment]
Looks grand.

**Comment by caseydavenport** [comment]
Huh.  Looks like I didn't actually address some of those comments.  should do that...

**Comment by tomdee** [comment]
Yes you should. But then just merge it. I don't need to look again.

**Pull request merged**